package.json

```diff
@@ -63,9 +64,12 @@
     "ajv": "^6.12.0",
     "amex-jest-preset": "^6.0.1",
     "eslint": "^6.2.2",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-prettier": "^3.1.2",
```

Comment:
Could we move these to `dependencies`? We'd need them available when anyone installs the `eslint-config-amex` or they will need to be `peerDependencies`.